### DISCUSSION TITLE
Cassandra plugin complex data types to varchar

### DISCUSSION THREAD

**Discussion by narayanbhawar10** [trinodb/trino#22316] [General]
Hi Team,
In cassandra plugin, most of the complex cassandra datatypes are mapped to trino's varchar datatypes. Is there any reason behind this. If we want to convert to proper datatypes for example map to map ,list to list then what procedure we need to follow. Is there any example available.

**Comment by ebyhr** [comment]
This is a known issue we want to fix. There's no way to change the mapping until we fix the connector. Please take a look at https://github.com/trinodb/trino/issues/660

**Reply by narayanbhawar10** [reply]
https://github.com/trinodb/trino/issues/660 This discussion is still open and inactive

**Reply by MR-GOYAL** [reply]
There seems to be some progress in this issue #660 . Is it closed without merge? By connector you means trino-cassandra plugin or cassandra driver?
